src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement.Api.UnitTests/Controllers/ProviderCourseDeleteControllerTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.RoatpCourseManagement.Api.Controllers;
+using SFA.DAS.RoatpCourseManagement.Application.Standards.Commands.DeleteProviderCourse;
+using SFA.DAS.Testing.AutoFixture;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.RoatpCourseManagement.Api.UnitTests.Controllers
+{
+    [TestFixture]
+    public class ProviderCourseDeleteControllerTests
+    {
+        [Test, MoqAutoData]
+        public async Task DeleteProviderCourse_InvokesCommand(
+            [Frozen] Mock<IMediator> mediatorMock,
+            [Greedy] ProviderCourseDeleteController sut,
+            int ukprn, int larsCode, DeleteProviderCourseCommand command)
+        {
+            await sut.DeleteProviderCourse(ukprn, larsCode, command);
+
+            mediatorMock.Verify(m => m.Send(command, It.IsAny<CancellationToken>()));
```

Comment:
You could combine this line into the test below. I don't mind it staying as is though, I'll leave the decision to you.

Reply:
One assert per test is best practise that we have not been following.